src/routes/api/oba/arrivals-and-departures-for-stop/[id]/+server.js

```diff
@@ -1,6 +1,8 @@
-import arrivalDepartureAPI from '$lib/RestAPI/arrivalsAndDeparturesForStop';
+import arrivalsAndDeparturesForStop from '$lib/RestAPI/arrivalsAndDeparturesForStop';
 
 /** @type {import('./$types').RequestHandler} */
+
 export async function GET({ params }) {
-	return arrivalDepartureAPI(params.id);
+	const stopID = params.id;
+	return arrivalsAndDeparturesForStop(stopID);
```

Comment:
import obaSdk and use that directly.

Reply:
I replied in the last review about the options we have.